using-timescaledb/data-retention.md

```diff
@@ -20,6 +26,68 @@ the `conditions` hypertable will still have data stretching back 36 hours.
 For more information on the `drop_chunks` function and related
 parameters, please review the [API documentation][drop_chunks].
 
+### Data Retention and Continuous Aggregates
+
+The function `drop_chunks` can be used to control data retention for
+the hypertable as well as the continuous aggregate. Given a hypertable
+`conditions` and a continuous aggregate `conditions_summary` over the
+hypertable:
+
+```sql
+CREATE VIEW conditions_summary WITH (timescaledb.continuous) AS
+SELECT location,
+       time_bucket('1d', "time") AS day,
+       AVG(temperature) AS avg_temperature,
+       MAX(temperature) AS max_temperature,
+       MIN(temperature) AS min_temperature
+FROM conditions
+GROUP BY location, time_bucket('1d', "time")
+```
+
+Data retention on a continous aggregate will normally not propagate to
+the hypertable, so the call below will only remove chunks from
+`conditions_summary`.
+
+```sql
+SELECT drop_chunks(INTERVAL '30 days', 'conditions_summary');
```

Comment:
Is `INTERVAL` really required here?

Reply:
Nope, but I think it is clearer.

Reply:
I tested just to make sure: the `INTERVAL` is needed here, a string will not do.